### DISCUSSION TITLE
I don't understand the green check marks on `update`

### DISCUSSION THREAD

**Discussion by gwww** [Homebrew/discussions#3328] [Everyday usage] [answered]
I understood when I do a `brew update` that anything marked with a green check mark is something that is outdated. The follow is the what I just got:
```
~ λ brew update
Updated Homebrew from a3fab022e to 68043d4e7.
Updated 4 taps (homebrew/core, homebrew/cask, homebrew/cask-fonts and homebrew/cask-drivers).
==> Updated Formulae
openssl@1.1 ✔                                              tree-sitter ✔                                              zoxide ✔
==> Updated Casks
alt-tab ✔

You have 1 outdated formula installed.
You can upgrade it with brew upgrade
or list it with brew outdated.
~ λ brew outdated
tree-sitter (0.20.6) < 0.20.6_1
```
Note the 3 updated formula but only 1 item outdated. A `brew upgrade` installed only `tree-sitter`. I'm guessing I don't understand the purpose of the check marks, but at least one other possibility is a bug.

**Comment by SMillerDev** [comment]
What does `brew doctor` and `brew config` say

**Comment by Bo98** [comment, marked as answer]
`openssl@1.1` and `zoxide` received changes, but the package version for them did not change as the changes were minor enough to not require upgrading (both were build-from-source fixes).

The tick is supposed to indicate formulae you have installed. We however have recently changed the default behaviour to only ever show installed formulae in that output so you shouldn't actually see anything without a tick. We could perhaps remove the tick for the new default behaviour.

**Comment by gwww** [comment]
Thx @Bo98. I like not showing only the installed formula! What I care about most is do I have anything to upgrade. So, for me the best output would be showing `outdated`. So in the case of what my initial question the output would have only `tree-sitter`.

Anyway I believe this is not a bug, which was my concern. Thanks!

**Comment by Bo98** [comment]
Based on this discussion, we've modified the default to now show a list of all outdated formulae and casks.

`HOMEBREW_UPDATE_REPORT_ALL_FORMULAE` is available for those who want the original behaviour we had for years.
